ethcore/sync/src/chain/supplier.rs

```diff
@@ -226,7 +226,7 @@ impl SyncSupplier {
 		let mut added_receipts = 0usize;
 		let mut data = Bytes::new();
 		for i in 0..count {
-			if let Some(mut receipts_bytes) = io.chain().block_receipts(&rlp.val_at::<H256>(i)?) {
+			if let Some(mut receipts_bytes) = io.chain().encoded_block_receipts(&rlp.val_at::<H256>(i)?) {
```

Comment:
IF we remove the other occurence of `encoded_block_receipts`, it seems that's the only one. I'd remove that method from a trait completely, and just use `.map(::rlp::encode(&x)`` here instead.